HOW/_requirements.md

```diff
@@ -0,0 +1,30 @@
+---
+HOW: <path in HOW tree>
+Source: <path to process used to create this HOW>
+Created: <YYYY-MM-DD>
+Version: <version-type(semantic/index)/version-number>
+Status: <initial status from process>
+---
+
+## Title
+The title of the standard being defined
```

Comment:
Is "standard" the correct term? I don't imagine there are going to be many conductor clones out there trying to build to these proposals. But I could be wrong!

Reply:
What I'm envisioning is that the HOW tree is the place where formal alignment happens, and a "standard" is a more normal name for "formal alignment".  There are two big classes of places where we will/should/could define technical standards, one of them is regarding:  

1) Holochain itself: i.e. the Kitsune Wire Protocol, or the conductor API, or various of the Holochain data formats from HRLs to the prefixes of the Hash types, etc. 

2) Things built on top of Holochain, like the [Joining Code format](https://github.com/holochain/HOW/pull/2) which I think of as a standard, or the Weave which should be a set of standards that would allow many people to create Weave "compliant" runtimes of which Moss is one.  Those standards are like the ones that web browsers implement: HTML, CSS, DOM, Javascript...

So this starting tree has `/tech/core` and `/tech/applications` to embody that.   I'm assuming that to start with we won't spend a lot of time filling out the `/tech/core` sections with existing APIs and specs, but, that we will for example make do the work of aligning on a standard for conductor services there.

Eventually, when someone actually wants to build the second conductor implementation then maybe then we do the full HOW alignment work on a the standards needed for that to happen.

Reply:
I agree about the term "standard". I expect to be able to implement a standard and then communicate/integrate with other things that implemented it. That way of thinking roughly agrees with the [definition of this term](https://en.wikipedia.org/wiki/Software_standard)

Are we explicitly limiting ourselves to this definition here? If we're going to go wider than that then I'd prefer a broader term and we can include the word "standard" in the title when we have that intention.

Reply:
I would need examples for processes or patterns or what else you have in mind, that would justify a deviation from the simple term "standard".